measures/BuildExistingModel/measure.rb

```diff
@@ -243,7 +249,11 @@ def run(model, runner, user_arguments)
     measures_dir = File.join(File.dirname(__FILE__), '../../measures')
     hpxml_measures_dir = File.join(File.dirname(__FILE__), '../../resources/hpxml-measures')
     lookup_file = File.join(resources_dir, 'options_lookup.tsv')
-    buildstock_csv_path = File.absolute_path(File.join(characteristics_dir, 'buildstock.csv')) # Should have been generated by the Worker Initialization Script (run_sampling.rb) or provided by the project
+
+    buildstock_csv_path = args['buildstock_csv_path']
+    unless (Pathname.new buildstock_csv_path).absolute?
+      buildstock_csv_path = File.absolute_path(File.join(characteristics_dir, buildstock_csv_path)) # Should have been generated by the Worker Initialization Script (run_sampling.rb) or provided by the project
```

Comment:
The buildstock_csv arg can already be specified by the resstock project file, right? So technically you could just use a bunch of different project files, one for each buildstock?
This does seem more convenient but I'm wondering if there's a different reason here that I'm missing.

Reply:
Yes. For us that means no more copy from run folder to the resstock/lib


The yaml has a sample file param. But run_analysis.rb just copied it. BuildExistingModel measure plain assumed it was in the resstock/lib folder